src/controllers/cartsController.js

```diff
@@ -27,4 +28,21 @@ const getCartsController = async (req, res) => {
   }
 };
 
-module.exports = { addItemToCartsController, getCartsController };
+const deleteProductController = async (req, res) => {
+  const { product_option_id } = req.query;
+  const userId = req.decoded;
+  try {
+    for (const ele of product_option_id) {
```

Comment:
```suggestion
    for (const ele of product_option_id) {
```
for문을 사용하시는것보니 id를 한개가 아니라 여러개를 받을 수 있나봅니다. 그럼 변수명도 그에 맞게 수정해주세요.